### PR TITLE
add test query item

### PR DESCRIPTION
**Pull request by GNaD13**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

File: x/pylons/keeper/grpc_query_get_stripe_refund_test.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	"github.com/Pylons-tech/pylons/x/pylons/types"
+)
+
+func (suite *IntegrationTestSuite) TestGetStripeRefund() {
+	goCtx := sdk.WrapSDKContext(suite.ctx)
+
+	correctAddr := "pylo1xn72u3jxlpqx8tfgmjf0xg970q36xensjngsme"
+	amount := sdk.NewIntFromUint64(10020060)
+	productID := "recipe/Easel_CookBook_auto_cookbook_2022_06_14_114716_442/Easel_Recipe_auto_recipe_2022_06_14_114722_895"
+	signature := "8lZsKTOdMuJSoFn0RCGEUGpPXl4YzLmhJMrEiAd4qZh99S4IIGbvcsXyOcOHdlKi6Yys9NhmkLN4LqSlq8Y1Cw=="
```
Comment (nghuyenthevinh2000) [inline x/pylons/keeper/grpc_query_get_stripe_refund_test.go:17]: how do you get this signature?

Comment (GNaD13) [reply x/pylons/keeper/grpc_query_get_stripe_refund_test.go:17]: I get from other test. In this test, this signature use to create the message for StripeRefund

> https://github.com/Pylons-tech/pylons/blob/main/x/pylons/keeper/msg_server_add_stripe_refund_test.go

Comment (nghuyenthevinh2000) [reply x/pylons/keeper/grpc_query_get_stripe_refund_test.go:17]: :v, just hate this kind of arbitrary signature.

**Review by mustafapylons — COMMENTED** [review]
As soon as CI is pass again, this can be re approved. Looks good to me.

**Comment by GNaD13** [comment]
> As soon as CI is pass again, this can be re approved. Looks good to me.

I have done with the CI. Can you approve my PR please @mustafapylons

**Review by faddat — APPROVED** [review]

**Pull request merged**